soh/src/overlays/actors/ovl_Door_Ana/z_door_ana.c

```diff
@@ -56,6 +56,26 @@ static ColliderCylinderInit sCylinderInit = {
     { 50, 10, 0, { 0 } },
 };
 
+static ColliderCylinderInit sUselessCylinderInit = {
```

Comment:
as with the member variables I think a name which better represenets what this is for would be good